python/lsst/obs/subaru/crosstalk.py

```diff
@@ -237,7 +246,7 @@ def subtractXTalk(mi, coeffs, minPixelToMask=45000, crosstalkStr="CROSSTALK"):
         mi.getMask().addMaskPlane(crosstalkStr)
         afwDisplay.getDisplay().setMaskPlaneColor(crosstalkStr, afwDisplay.MAGENTA)
         fs.setMask(mi.getMask(), crosstalkStr)  # the crosstalkStr bit will now be set whenever
-                                                # we subtract crosstalk
+        # we subtract crosstalk
```

Comment:
I put that comment on the line just above setMask: it's all on one line now, and the whole thing is shorter!